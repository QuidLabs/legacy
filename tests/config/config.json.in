{
    "setup": {
        "eosio": true,
        "vigor": {
            "vigorTokenContract": "create",
            "vigorLendingContract": "create",
            "tokenContracts": "create",
            "oracleContract": "create",
            "dataPreProcContract": "create",
            "contractAccounts": true,
            "feederAccounts": true,
            "testAccounts": true
        },
        "oracles": {
            "waitForFeeders": true
        }
    },
    "walletPath" : "wallet",
    "system" : {
      "accounts": ["eosio.bpay", "eosio.msig", "eosio.names", "eosio.ram", "eosio.ramfee", "eosio.saving", "eosio.stake", "eosio.token", "eosio.vpay", "eosio.rex"],
      "stakedAccounts": ["eosio.wrap"],
      "contracts": ["eosio.msig", "eosio.token", "eosio.system", "eosio.bios"],
      "contractsPath": "@EOSIO_CONTRACTS_PATH@"
    },
    "user" : {
      "accounts" : [],
      "testAccounts" : ["finalreserve", "vigordacfund", "account11111", "account11112", "account11113", "account11114", "account11115", "account11121", "account11122", "account11123"],
      "contractAccounts": ["vigorlending", "vigortoken11", "vig111111111", "dummytokensx", "datapreproc2", "oracleoracl2"],
      "feederAccounts": ["feeder111111", "feeder111112", "feeder111113"],
      "contracts" : ["vigorlending", "eosio.token", "dummytokensx", "vig111111111", "datapreproc2"],
      "contractsPath": "@VIGOR_CONTRACTS_PATH@"
    },
    "data": [{
          "url": "https://api.newdex.io/v1/price?symbol=vig111111111-vig-eos",
          "interval": 60000,
          "contract": "oracleoracl2",
          "pair": "vigeos",
          "calculator": "parseInt(Math.round(jsonData.data.price * 1000000))",
          "fixed": "parseInt(Math.round(0.000318 * 1000000))",
          "permission": "active"
      },{
          "url": "https://api.newdex.io/v1/price?symbol=boidcomtoken-boid-eos",
          "interval": 60000,
          "contract": "oracleoracl2",
          "pair": "boideos",
          "calculator": "parseInt(Math.round(jsonData.data.price * 1000000))",
          "fixed": "parseInt(Math.round(0.0000343 * 10000000))",
          "permission": "active"
      },{
          "url": "https://api.newdex.io/v1/price?symbol=everipediaiq-iq-eos",
          "interval": 60000,
          "contract": "oracleoracl2",
          "pair": "iqeos",
          "calculator": "parseInt(Math.round(jsonData.data.price * 1000000))",
          "fixed": "parseInt(Math.round(0.000272 * 1000000))",
          "permission": "active"
      },{
          "url": "https://min-api.cryptocompare.com/data/price?fsym=EOS&tsyms=USD&e=coinbase",
          "interval": 60000,
          "contract": "oracleoracl2",
          "pair": "eosusd",
          "calculator": "parseInt(Math.round((jsonData.USD) * 10000))",
          "fixed": "parseInt(Math.round((3.733) * 10000))",
          "permission": "active"
    }],
    "tests": {
        "resultsBasePath": "@EXPECTED_RESULTS_PATH@",
        "stress": {
            "enabled": false,
            "accountGeneration": {
                "prefix": "aaaaaaaaa",
                "amount": 1000
            }
        },
        "basic": {
            "enabled": true
        },
        "fixedFeeders": true,
        "contractInTestMode": true
    }
}
