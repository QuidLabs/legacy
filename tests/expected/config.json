{
  "rows": [{
      "alphatest": 900,
      "soltarget": 10,
      "lsoltarget": 25,
      "maxtesprice": 5000,
      "mintesprice": 50,
      "calibrate": 10,
      "maxtesscale": 20,
      "mintesscale": 1,
      "reservecut": 20,
      "savingscut": 10,
      "maxlends": 30,
      "freezelevel": 0,
      "assetouttime": 2,
      "initialvig": 0,
      "viglifeline": 3,
      "vigordaccut": 10,
      "newacctlim": 10,
      "newacctsec": 86400,
      "reqstake": 100,
      "staketime": 259200,
      "repanniv": 2592000,
      "maxdisc": 25,
      "exectype": 2,
      "minebuffer": 30,
      "gasfee": 1,
      "kickseconds": 1200000,
      "initmaxsize": 50000,
      "assetintime": 3,
      "debtceiling": 250000,
      "logcount": 3000,
      "gatekeeper": 0,
      "liquidate": 1,
      "accountslim": 100,
      "mincollat": 111,
      "rexswitch": 100,
      "dataa": 2000000,
      "datab": 1000000,
      "datac": 100,
      "datad": "extrafield11",
      "proxycontr": "vigorrewards",
      "proxypay": "genereospool",
      "dactoken": "dactoken1111",
      "oraclehub": "vigoraclehub",
      "daccustodian": "daccustodia1",
      "vigordacfund": "vigordacfund",
      "finalreserve": "finalreserve",
      "bailoutcr": 110,
      "bailoutupcr": 110,
      "ratesub": 0,
      "liqcut": 50,
      "txnfee": 1,
      "minerfee": 1,
      "lpsxcut": 10,
      "minc": 5,
      "tickfee": 1
    }
  ],
  "more": false,
  "next_key": ""
}
